spec/requests/api/arbitration_rule_spec.rb

```diff
@@ -0,0 +1,112 @@
+RSpec.describe 'Arbitration Rule API' do
+  context 'arbitration rules index' do
+    it 'rejects requests without an appropriate role' do
+      api_basic_authorize
+
+      run_get arbitration_rules_url
+
+      expect(response).to have_http_status(:forbidden)
+    end
+
+    it 'can list arbitration rules' do
+      rules = FactoryGirl.create_list(:arbitration_rule, 2)
+      api_basic_authorize collection_action_identifier(:arbitration_rules, :read, :get)
+
+      run_get arbitration_rules_url
+
+      expect_result_resources_to_include_hrefs(
+        'resources',
+        rules.map { |rule| arbitration_rules_url(rule.id) }
+      )
+      expect(response).to have_http_status(:ok)
+    end
+  end
+
+  context 'arbitration rules create' do
+    let(:request_body) do
+      {
+        'name'       => 'admin rule',
+        'operation'  => 'inject',
+        'expression' => {
```

Comment:
@gtanzillo could there be an issue with allowing arbitrary MiqExpressions to be created through the API?